modules/client-configuration/pages/delete-clients.adoc

```diff
@@ -92,33 +92,45 @@ Instead of [command]``zypper`` use equivalent packager commands such as [command
 
 .Procedure: Deleting SLES 12 and 15 Clients
 . Stop the salt-minion service:
+
 +
+
+[source,shell]
 ----
 systemctl stop salt-minion
 ----
 +
 . Remove the repositories and configuration files:
+
 +
+
+[source,shell]
 ----
-rm /etc/zypp/repos.d/susemanager\:channels.repo
+rm /etc/zypp/repos.d/susemanager\:bootstrap.repo
```

Comment:
I think we need to both. susemanager:channels.repo exists when this system was successful registered and not cleaned up.

So I would suggest to use "-f" to prevent errors when the file does not exist.

```
rm -f /etc/zypp/repos.d/susemanager\:channels.repo
rm -f /etc/zypp/repos.d/susemanager\:bootstrap.repo
```